assets/vue/components/BlockedIncident.vue

```diff
@@ -51,16 +52,30 @@ export default {
         const {version} = value.linkinfo;
         const {groupid} = value.linkinfo;
         const newkey = `${groupid}:${version}`;
-        if (!(newkey in results)) {
-          results[newkey] = {name: value.name, passed: 0, failed: 0, stopped: 0, waiting: 0};
-          results[newkey].linkinfo = value.linkinfo;
-          results[newkey].linkinfo.flavor = [];
+        if (this.groupNames.toLowerCase().split(',').includes(value.name.toLowerCase()) || this.groupNames == '') {
+          if (!(newkey in results)) {
+            results[newkey] = {name: value.name, passed: 0, failed: 0, stopped: 0, waiting: 0};
+            results[newkey].linkinfo = value.linkinfo;
+            results[newkey].linkinfo.flavor = [];
+          }
+          results[newkey].linkinfo.flavor.push(flavor);
```

Comment:
I'd really avoid the repeated access via `results[newkey]`. Just use a variable.

Reply:
Aside from being more readable, it is actually faster too.